### PR TITLE
Updated URL and query parameters

### LINKED ISSUE #14: 401 error - failing to set up component
The Min renovasjon integration stopped working today. I get 401 errors in the log.

image

Here is a copy from the log:
Logger: homeassistant.setup
Source: setup.py:288
First occurred: 22:16:10 (1 occurrences)
Last logged: 22:16:10

Error during setup of component min_renovasjon
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 288, in _async_setup_component
    result = await task
             ^^^^^^^^^^
  File "/usr/local/lib/python3.11/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/min_renovasjon/__init__.py", line 57, in setup
    min_renovasjon = MinRenovasjon(
                     ^^^^^^^^^^^^^^
  File "/config/custom_components/min_renovasjon/__init__.py", line 72, in __init__
    self._kalender_list = self._get_calendar_list()
                          ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/min_renovasjon/__init__.py", line 138, in _get_calendar_list
    kalender_list = self._parse_calendar_list(tommekalender, fraksjoner)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/min_renovasjon/__init__.py", line 154, in _parse_calendar_list
    tommekalender_json = json.loads(tommekalender)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/json/__init__.py", line 339, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType

Im not that familiar with HTTP-request responsens but doesnt 401 mean that there is something wrong with authentication?

### PR DESCRIPTION
**Pull request by husjon**
As discussed in #14 there have been changes to the API, it is now behind an Azure proxy.
This PR implements the changes by @TomTorger

Fixes #14

### CODE REVIEW

File: custom_components/min_renovasjon/__init__.py [hunk 1]
```diff
@@ -22,11 +22,13 @@
 CONST_KOMMUNE_NUMMER = "Kommunenr"
 CONST_APP_KEY = "RenovasjonAppKey"
 CONST_URL_FRAKSJONER = (
-    "https://komteksky.norkart.no/komtek.renovasjonwebapi/api/fraksjoner"
+    "https://norkartrenovasjon.azurewebsites.net/proxyserver.ashx?server="
+    "https://komteksky.norkart.no/MinRenovasjon.Api/api/fraksjoner/"
```
Comment (Danielhiversen) [inline custom_components/min_renovasjon/__init__.py:26]: Is the intention to keep the old URL?

Comment (Danielhiversen) [inline custom_components/min_renovasjon/__init__.py:26]:
```suggestion
    "https://norkartrenovasjon.azurewebsites.net/proxyserver.ashx?server="
```

**Review by Danielhiversen — APPROVED** [review]
Great 👍

**Comment by erazor666** [comment]
Do we need to add kommunenr to our config for this fix to work?

**Comment by husjon** [comment]
@erazor666 yeah the kommunenr is now required after the changes to the API.
You can find your kommunenr by following the Readme under the `street_code` and `county_id` section.

**Pull request merged**